tests/common/type_coverage.h

```diff
@@ -439,6 +496,50 @@ void for_all_types_vectors_marray(const named_type_pack<types...> &typeList,
   assert((typeNameIndex == sizeof...(types)) && "Pack expansion failed");
 }
 
+/**
+ * @brief Run action for std device_copyable containers with type T
+ * @tparam action Functor template for action to run
+ * @tparam actionArgsT Parameter pack to use for functor template instantiation
+ * @tparam argsT Deduced parameter pack for arguments to forward into the call
+ * @param args Arguments to forward into the call
+ */
+template <template <typename, typename...> class action, typename T,
+          typename... actionArgsT, typename... argsT>
+void for_device_copyable_std_containers(argsT &&...args) {
+  for_all_types<action, actionArgsT...>(
+      type_pack<std::array<T, 5>, std::optional<T>, std::pair<T, T>,
```

Comment:
Why 5? Can we avoid using magic numbers?

Reply:
Magic number has been removed in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/358/commits/064e7d2b8871bf44c60f2ca7a9b11e2653f414dc